tests/test_find_people_to_vaccinate.py

```diff
@@ -34,14 +36,17 @@ def test_find_people_to_vaccinate_no_refusers(params):
     # 0.35 is vaccinated this period. 0.45 and above not yet.
     expected = pd.Series([True] + [False] * 5)
 
-    res = find_people_to_vaccinate(
-        receives_vaccine=None,
-        states=states,
-        params=params,
-        seed=33,
-        vaccination_shares=vaccination_shares,
-        init_start=pd.Timestamp("2021-01-15"),
-    )
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore")
```

Comment:
Do you really want to ignore all warnings?

Reply:
I changed it to ignoring all UserWarnings. If I try to include a message the warnings always appear in pytest. :shrug: